src/chains/ethereum/ethereum/src/forking/trie.ts

```diff
@@ -94,15 +103,18 @@ export class ForkTrie extends GanacheTrie {
     startBlockNumber: Quantity,
     endBlockNumber: Quantity
   ) {
-    const db = this.metadata._leveldb;
-    const stream = db.createKeyStream({
+    //@ts-ignore
+    const db = this.metadata.db._leveldb;
```

Comment:
```suggestion
    const db = (this.metadata.db as any)._leveldb;
```
I prefer this, as @ts-ignore is very broad